src/main/java/frc/robot/subsystems/elevatorIO/ElevatorIOSparks.java

```diff
@@ -87,11 +93,36 @@ public void updateInputs(ElevatorInputs inputs) {
     inputs.velocityInchesPerSecondRight = right.getEncoder().getVelocity();
     inputs.tempCelsiusRight = right.getMotorTemperature();
     inputs.currentDrawAmpsRight = right.getOutputCurrent();
+
+    currentLeftFF = leftFF.calculate(inputs.velocityInchesPerSecondLeft);
```

Comment:
Previously, we used the feedforward as part of our _velocity_ control loop - on the rio, we were generating a list of velocity setpoints for the motors to "chase," which if followed correctly, would land us in the right position. As a result, the feedforward needed to be based on our setpoint's velocity (not the current velocity like you have here).

We are not doing that on the Sparks. The equivalent on the Sparks would be `CANSparkMax.ControlType.kSmartMotion`, but it has some issues and we should stick with `CANSparkMax.ControlType.kPosition` (which is what you have). The difference is that the feedforward now needs to be based on the setpoint position, not the setpoint velocity.

You should set `currentLeftFF = leftFF.calculate(targetHeight);` in `goToSetpoint()`.

Reply:
Fixed in 5fe8021b74669a15333238740e561b012b0ebede